flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/StreamExecutionEnvironment.java

```diff
@@ -2330,4 +2330,8 @@ public void registerCachedFile(String filePath, String name, boolean executable)
 		}
 		return (T) resolvedTypeInfo;
 	}
+
+	private String getJobName() {
+		return configuration.getString(PipelineOptions.NAME, DEFAULT_JOB_NAME);
```

Comment:
Can't we just put the default name in the pipeline option? That way, it will also be rendered in the documentation.

Reply:
Are pipeline options also used in `ExecutionEnvironment`? The PR doesn't update this class but maybe it should? In that case `Flink Streaming Job` would be wrong.

Reply:
Thanks all for the suggestions. @twalthr, I have checked that the pipeline options are also used in `ExecutionEnvironment`. I also don't notice that before, I will update it. The default name of `ExecutionEnvironment` is `"Flink Java Job at " + Calendar.getInstance().getTime()`, so the `NAME` option has no default value.

Reply:
`TableEnvironment` also uses a different default name for the SQL query, e.g. "insert-into_<sink-names>". So I think no default value makes more sense, that the default value is defined by different environment.